components/Common/AvatarGroup/Avatar/index.tsx

```diff
@@ -0,0 +1,25 @@
+import * as RadixAvatar from '@radix-ui/react-avatar';
+import type { FC } from 'react';
+
+import type { AvatarType } from '@/components/Common/AvatarGroup';
+
+import styles from './index.module.css';
+
+type AvatarProps = AvatarType;
+
+export const fallbackAltAsAvatar = (alt: string = '') =>
```

Comment:
As I was saying, this should probably be removed from here, so you only render `alt`

And exported to the same GitHub utility or a string utility (I explained the thing about being a generic function that can create these acronyms with X characters:)

```tsx
export const getAcronymFromString = (str = '', length = 2) =>
 str.match(/\b(\w)/g).join('').toLocaleUpperCase().substr(0, length);
```

And then that's called by AvatarGroup, for example.

Reply:
(and yea this should not be exported from here 🙈)